src/main/java/seedu/address/model/time/TimePeriod.java

```diff
@@ -22,8 +22,8 @@ public abstract class TimePeriod {
      * Constructs a period in time.
      */
     public TimePeriod(LocalTime startTime, LocalTime endTime, Day schoolDay) {
-        if (startTime.isAfter(endTime)) {
-            throw new WrongTimeException("Start Time Cannot be after End Time!");
+        if (startTime.isAfter(endTime) || startTime.isEqual(endTime)) {
```

Comment:
Alright. Noted. Though a simpler implementation would just be:
```suggestion
        if (endTime.isAfter(startTime)) {
```

Reply:
You forgot to negate the predicate